org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -560,6 +574,13 @@
             properties="isNodeResource"
             type="java.lang.Object">
       </propertyTester>
+      <propertyTester
+            class="org.eclipse.wildwebdeveloper.debug.firefox.Tester"
+            id="org.eclipse.wildwebdeveloper.propertyTester.isHTMLProject"
```

Comment:
Should be more something like `isHTMLLaunchable` so we can reuse it for Chrome support later.